rust/stub/src/uefi_helpers.rs

```diff
@@ -1,12 +1,180 @@
 use core::ffi::c_void;
```

Comment:
The changes in this module deserve their own module: maybe `evifars.rs`?

Reply:
Agreed